README.md

```diff
@@ -1,3 +1,5 @@
+![nuxt-auth demo page](.github/nuxt-auth.jpg)
+
 # 🔐 nuxt-auth
 
 [![npm version][npm-version-src]][npm-version-href]
```

Comment:
broken link below detected: https://sidebase.io/nuxt-auth/quick-start

can you fix it? should be: https://sidebase.io/nuxt-auth/getting-started/quick-start